nat-lab/tests/config.py

```diff
@@ -59,15 +59,21 @@ def get_root_path(path: str) -> str:
     "ipv4": "10.0.100.1",
     "port": 51820,
     "public_key": "N2Kuejq5Gd553kGYKY4iwtnCuk9MJjAPpg7C04FOdDw=",
+    "private_key": "IK88LLAc0zr0iEGZ3bZaXMYb2nFFNUBYOpEOyL6vtl0=",
```

Comment:
Even though we used a shared key between two VPN servers previsouly, could we add different private keys to both testing VPN servers (this will have to change corresponding public keys in the clients code)?

I think this key sharing may have unintended consequences, and introduce some flakyness due to WG roaming.

Reply:
also added key generation